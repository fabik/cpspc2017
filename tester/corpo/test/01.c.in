100
18 11 10 48 26 13 59 37 43 82 57 56 74 4 71 8 28 61 76 75 65 77 23 61 15 14 46 39 69 51 99 62 25 55 30 95 45 90 96 76 2 67 56 20 84 29 19 94 35 24 87 21 7 40 11 83 19 32 16 47 79 30 63 52 55 49 75 22 1 50 60 38 31 42 81 78 6 12 78 64 68 48 91 63 32 27 54 18 9 92 17 65 33 60 70 26 52 73 39
66 22 91 98 49 62 32 8 41 71 29 97 9 45 88 68 94 31 92 3 10 5 76 85 83 43 2 60 95 11 12 88 25 28 47 33 25 87 82 53 55 90 74 64 86 56 25 33 35 67 56 73 19 62 4 71 90 67 50 21 96 14 87 13 26 68 78 99 45 44 65 30 69 45 72 61 24 89 36 6 94 20 1 14 84 67 40 54 38 81 84 92 43 79 23 42 77 80 70
