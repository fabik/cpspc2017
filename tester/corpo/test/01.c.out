99 14 0 39 0 39 0 18 71 23 4 61 0 38 0 0 0 0 34 27 0 12 14 8 1 53 0 16 71 15 22 20 46 0 83 0 0 21 0 63 0 3 48 0 1 0 45 0 45 82 0 0 0 0 20 4 0 0 0 79 23 18 0 2 7 0 7 0 20 1 43 0 10 24 0 5 16 8 30 0 21 8 66 19 0 0 3 0 0 63 13 19 0 49 34 23 0 0 14 0 
